tests/Zend/Ldap/SearchTest.php

```diff
@@ -162,6 +162,11 @@ public function testSearchNothingGetFirst()
 
     public function testSorting()
     {
+        if(PHP_VERSION_ID >= 70000)
+        {
+            return true;
```

Comment:
this test should be marked as skipped, with a comment why, instead of returning true.